tests/scripts/make-counts.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+while getopts p:l: option
+do
+case "${option}"
+in
+p) INFO_PATH=${OPTARG:-.};;
+l) LOG=${OPTARG:-$(date)};;
+esac
+done
+
+ls $INFO_PATH/*.info | xargs -I {} sh -c "cat {} | jq .status > {}.counts" | tee $LOG.log
```

Comment:
These files are placed in the `tests/` folder but I don't see any automated testing being done that is using these files.